torch/csrc/jit/pickler.cpp

```diff
@@ -1,5 +1,6 @@
 #include <ATen/ATen.h>
 #include <ATen/core/Dict.h>
+#include <torch/csrc/distributed/rpc/rref_context.h>
```

Comment:
So far the build and test do not complain, but if there is another approach to avoid dependency on rref_context, that is even better, will follow up with you offline